pywr/notebook/__init__.py

```diff
@@ -153,13 +153,25 @@ def pywr_json_to_d3_json(model):
 
     nodes = []
     node_classes = create_node_class_trees()
+    params = [param for param in model["parameters"]]
     for node in model["nodes"]:
-        json_node = {'name': node["name"], 'clss': node_classes[node["type"].lower()]}
+        json_node = {'name': node.pop("name"), 'clss': node_classes[node["type"].lower()]}
         try:
             json_node['position'] = node['position']['schematic']
         except KeyError:
             pass
 
+        if attributes:
+            json_node["attributes"] = []
+            for name, val in node.items():
+                val_type = " "
```

Comment:
Perhaps this should be `None` by default.

Reply:
I've used an empty string as `None` causes an error when loading the data in js